frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/utils/behavior-to-german-name.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { ExerciseSimulationBehaviorState } from 'digital-fuesim-manv-shared';
+
+const behaviorToGermanNameDictionary = {
+    assignLeaderBehavior: 'Führung zuweisen',
+    treatPatientsBehavior: 'Patienten behandeln',
+    unloadArrivingVehiclesBehavior: 'Fahrzeuge entladen',
+};
```

Comment:
I would add typings to this. Though you already have type-safety because of the very simple pipe-implementation

```suggestion
const behaviorToGermanNameDictionary: {
    [Key in ExerciseSimulationBehaviorState['type']]: string;
} = {
    assignLeaderBehavior: 'Führung zuweisen',
    treatPatientsBehavior: 'Patienten behandeln',
    unloadArrivingVehiclesBehavior: 'Fahrzeuge entladen',
};
```